app/src/sharedTest/java/org/oppia/android/app/story/StoryFragmentTest.kt

```diff
@@ -306,6 +326,63 @@ class StoryFragmentTest {
     }
   }
 
+  @Test
+  fun testStoryFragment_chapterMissingPrerequisiteThumbnailIsBlurred() {
+    launch<StoryActivity>(createFractionsStoryActivityIntent()).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(allOf(withId(R.id.story_chapter_list))).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          2
+        )
+      )
+
+      testCoroutineDispatchers.advanceUntilIdle()
+      onView(
+        atPositionOnView(
+          R.id.story_chapter_list,
+          2,
+          R.id.chapter_thumbnail
+        )
+      ).check { view, noViewFoundException ->
+        Mockito.verify(view.chapter_thumbnail.imageLoader, times(2)).loadDrawable(
+          anyInt(),
+          anyOrNull(),
+          capture(listCaptor)
+        )
+        assertThat(ImageTransformation.BLUR).isIn(listCaptor.value)
+      }
+    }
+  }
+
+  @Test
+  fun testStoryFragment_changeConfiguration_chapterMissingPrerequisiteThumbnailIsBlurred() {
+    launch<StoryActivity>(createFractionsStoryActivityIntent()).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(isRoot()).perform(orientationLandscape())
+      onView(allOf(withId(R.id.story_chapter_list))).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
+          2
+        )
+      )
+
+      testCoroutineDispatchers.advanceUntilIdle()
+      onView(
+        atPositionOnView(
+          R.id.story_chapter_list,
+          2,
+          R.id.chapter_thumbnail
+        )
+      ).check { view, noViewFoundException ->
+        Mockito.verify(view.chapter_thumbnail.imageLoader, times(2)).loadDrawable(
```

Comment:
Nit: rather than ``times(2)`` we can probably use ``atLeastOnce()`` which will make the test a bit more robust (since it seems like we don't actually care what the exact number is--that's an implementation detail in this case).

Ditto above.

Reply:
Done, and it works.